python/metatomic_torch/tests/ase_calculator.py

```diff
@@ -549,3 +550,22 @@ def test_additional_outputs(atoms):
 
     another = atoms.calc.additional_outputs["another::one"]
     assert another.block().properties.names == ["another"]
+
+
+def test_lazy_loading_module():
+    """Tests that the `ase_calculator` module is lazy-loaded correctly."""
+    import metatomic.torch
+
+    module_name = "metatomic.torch.ase_calculator"
+    ase_module = metatomic.torch.ase
+    assert module_name in sys.modules
+    assert isinstance(ase_module, types.ModuleType)
```

Comment:
Wouldn't this work just because other parts of this file are importing this code? It might be worth moving this to a separate file for testing.